CompilerSource/settings-parse/crawler.cpp

```diff
@@ -147,8 +147,10 @@ namespace extensions
     cout << "\n\n\n\nStarting platform inspection\n";
     for (string ef = file_find_first("ENIGMAsystem/SHELL/Platforms/*",fa_sysfile | fa_readonly | fa_directory | fa_nofiles); ef != ""; ef = file_find_next())
     {
-      cout << " - ENIGMAsystem/SHELL/Platforms/" + ef + "/Info/About.ey: ";
-      ifstream ext(("ENIGMAsystem/SHELL/Platforms/" + ef + "/Info/About.ey").c_str(), ios_base::in);
+      const string ef_path = "ENIGMAsystem/SHELL/Platforms/" + ef + "/Info/About.ey";
+      if (!file_exists(ef_path)) continue;
```

Comment:
If you want to skip directories, why are you explicitly asking it to include them on line 148?

Reply:
Read my `tl;dr` if you want the answer to that, your file find ignores attributes in the Win32 version.

Reply:
Ok, so after Discord discussion I've made more changes: 6d9d01cee451dc7f2bf3a0decdbba08aca892848

I kept the `&` fix for the file attributes even though we don't need it now. It does mean that `platforms_mandatory.h/Info/About.ey` is not looked for though.

Finally, I've decided to just remove the failed messages which you guys said should not occur normally anyway. I decided to keep the opened messages though because they are helpful.